WBooks/TableBooks/TableBookController.swift

```diff
@@ -38,6 +39,18 @@ final class TableBookController: UITableViewController {
         tableBookView.booksTable.register(nib, forCellReuseIdentifier: CellBookView.identifier)
     }
     
+    func loadBooks() {
+        tableBookViewModel.fetchBook(onSuccess: self.reloadTable, onError: self.showError)
```

Comment:
Use closures with `[weak self]`:

```Swift
tableBookViewModel.fetchBook(onSuccess: { [weak self] in
   self?.reloadTable()
}, onError: { [weak self] in
   self?.showError()
})
```

We'll explain more about this in the [ARC card](https://trello.com/c/2V0Qxr3k/32-arc-automatic-reference-counting).